src/utilities/core-utils.test.ts

```diff
@@ -91,6 +91,127 @@ describe("CoreUtils", () => {
 
     // #endregion getRandomEnum
 
+    // -----------------------------------------------------------------------------------------
+    // #region numericEnumToPojo
+    // -----------------------------------------------------------------------------------------
+
+    describe("numericEnumToPojo ", () => {
+        test("when given numeric enum, returns an object of values", () => {
+            // Arrange
+            enum testEnum {
+                first,
+                second,
+                third,
+            }
+
+            // Act
+            const result = CoreUtils.numericEnumToPojo(testEnum);
+            const values = CoreUtils.objectToArray(result);
```

Comment:
If we can use built-in language features, functions or matchers in order to test, let's try to use those before our own functions.

Could this be written something along the lines of - 
```JS
expect(result.first).toBe(testEnum.first);
expect(result.second).toBe(testEnum.second);
expect(result.third).toBe(testEnum.third);
expect(Object.keys(result)).toHaveLength(3);
```

Reply:
`result.first` and so on will give an error for property doesn't exist. I will use `Object.keys` to compare the value

Reply:
Ah, it probably needs to be typed as `any`. I'm wondering why the function is typed to return `{}`